tests/jballerina-unit-test/src/test/resources/test-src/object/object_langlib_function_invocation_negative.bal

```diff
@@ -0,0 +1,41 @@
+import ballerina/lang.'object as lang_obj;
```

Comment:
Missing the licence header.

Reply:
Added in 1b6b21ff9a8ad716b114fd76ad04f19dc70a57e8